ublox/src/ubx_packets/cfg_val.rs

```diff
@@ -380,6 +380,15 @@ macro_rules! cfg_val {
           self.len() == 0
       }
 
+      #[inline]
+      pub const fn key(&self) -> CfgKey {
```

Comment:
I know not everything is equally well documented, but we're trying to improve it gradually :smile: so could you please add a doc comment, however trivial it might be.